packages/utilities/psammead-test-helpers/src/index.js

```diff
@@ -4,8 +4,13 @@ import deepClone from 'ramda/src/clone';
 
 export const shouldMatchSnapshot = (title, component) => {
   it(title, () => {
+    // Components with a single child do not have to be wrapped in a div when matching against a snapshot.
```

Comment:
I think a more appropriate comment would be something like

`// select the first child to remove the pointless wrapping div from snapshots`